{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "a8f3dae1_4281d454",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 4
      },
      "lineNbr": 0,
      "author": {
        "id": 1006192
      },
      "writtenOn": "2022-04-11T13:51:08Z",
      "side": 1,
      "message": "@Sven can you verify this manually? (I can\u0027t see the reason why the Zuul CI failed)",
      "revId": "44b5333d08d02c15334f81906134a5613d3f0a3b",
      "serverId": "173816e5-2b9a-37c3-8a2e-48639d4f1153"
    }
  ]
}